README.md

```
@@ -1,23 +1,41 @@
-## Code of Conduct
-We follow the [Code of Conduct](code-of-conduct.md) of the [CircuitVerse](https://circuitverse.org) Community.
 
-## Setup
-See [`SETUP.md`](SETUP.md) for more information on setting up CircuitVerse.
+## Configuration
+To configure the translation API endpoint for production:
 
-## Contributing
-See [`CONTRIBUTING.md`](CONTRIBUTING.md) for more information on contributing to CircuitVerse.
+Set the TRANSLATE_API_ENDPOINT environment variable:
+TRANSLATE_API_ENDPOINT=https://your-production-api-endpoint.com/translate
```

Markdownlint (0.35.0)

Bare URL used

(MD034, no-bare-urls)